website/docs/d/en_subscription_fcm.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: 'Event Notifications'
+layout: 'ibm'
+page_title: 'IBM : ibm_en_subscription_android'
+description: |-
+  Get information about a FCM subscription
+---
+
+# ibm_en_subscription_fcm
+
+Provides a read-only data source for subscription. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example usage
+
+```terraform
+data "ibm_en_subscription_android" "fcm_subscription" {
+  instance_guid = "my_instance_id"
```

Comment:
In all examples can you point some valid reference 
instance_guid = ibm_resource_instance.en_subscription_resource.guid
subscription_id = ibm_en_subscription_email.en_subscription_resource_4.subscription_id

somethig like this in all examples

Reply:
I have changed the variable references for all resources and data sources in examples. Please check.